### ISSUE TITLE
Element moves inside a paper-dialog

### ISSUE THREAD

**Issue by alan-strohm** [Polymer/paper-progress#20]
See: http://jsfiddle.net/fe4p2tha/

The progress bar element itself (not just the progress part) moves back and forth inside the dialog.  The same thing doesn't happen outside a dialog.

**Comment by nicolethenerd** [comment]
Am seeing the same behavior - any updates?

**Comment by nicolethenerd** [comment]
Wrapping the paper-progress in a div seems to be an ok workaround.

**Comment by keanulee** [comment]
Right, `<paper-dialog>` [adds padding](https://github.com/PolymerElements/paper-dialog-behavior/blob/master/paper-dialog-shared-styles.html#L48) to its contents (which `<paper-progress>` does not expect), so wrapping it with a `<div>` is suggested.
